maup/repair.py

```diff
@@ -166,6 +167,23 @@ def crop_to(source, target):
 
     return cropped_geometries
 
+def expand_to(source, target):
+    """
+    Expands the source geometries to the target geometries.
+    """
+    geometries = get_geometries(source).simplify(0).buffer(0)
+
+    source_union = unary_union(geometries)
+
+    leftover_geometries = get_geometries(target).apply(lambda x: x - source_union)
+    leftover_geometries = leftover_geometries[~leftover_geometries.is_empty].explode()
+
+    geometries = absorb_by_shared_perimeter(
+        leftover_geometries, get_geometries(source), relative_threshold=None
```

Comment:
why not pass in your `geometries` variable that has been simplified and buffered here for the second argument rather than `get_geometries(source)`?

Reply:
I feel that's out of scope of this function. We shouldn't tamper with geometries that don't need to be expanded. The user won't be expecting it, so I think we shouldn't do it. If they want to simplify/buffer, they can call the other functions in `repair.py` that will explicitly do that.